CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
```

Comment:
seems like you might need to run `prettier`, not sure why the pre-commit hook didn't format files correctly.